src/front-door/azext_front_door/_params.py

```diff
@@ -50,11 +59,14 @@ def load_arguments(self, _):
         c.argument('frontend_host_name', help='Domain name of the frontend endpoint.')
 
     with self.argument_context('network front-door', arg_group='HTTPS') as c:
-        c.argument('certificate_source', arg_type=get_enum_type(FrontDoorCertificateSource), help='Certificate source to enable HTTPS. Allowed values: FrontDoor, AzureKeyVault. For Azure Key Vault, right permissions need to be set for Front Door to to access the Key vault. Learn more at https://aka.ms/FrontDoorCustomDomain.')
+        c.argument('certificate_source', arg_type=get_enum_type(FrontDoorCertificateSource), help='Certificate source to enable HTTPS. Allowed values: FrontDoor, AzureKeyVault. For Azure Key Vault, right permissions need to be set for Front Door to access the Key vault. Learn more at https://aka.ms/FrontDoorCustomDomain.')
```

Comment:
This is going to display the allowed values twice, once because it is in your help string, and once because of the `get_enum_type`. I would remove the "Allowed values" annotation, and move the extended description "For Azure Key vault..." to a long description using the `help.py` file.

Reply:
Fixed. Moved the long summary to command 'enable-https' help as that seems the appropriate place